start-here.mdx

```diff
@@ -1,8 +1,153 @@
+</Aside>
+
+_TON_ is a [blockchain](https://en.wikipedia.org/wiki/Blockchain). It provides a distributed platform for hosting applications. Roughly speaking, it works as if it were a single server executing all the code. Such applications are called _smart-contracts_.
+
+The platform runs on a set of servers, called _validators_, owned by individuals or organizations with a large stake in TON and [greatly interested](/ecosystem/staking/overview) in keeping the platform safe, fair, and operational.
+
+_Toncoin_ is TON's primary [cryptocurrency](https://en.wikipedia.org/wiki/Cryptocurrency). It is used to pay for the execution of smart contracts, the storage of their data, and network traffic. Such payments are called [fees](/foundations/fees).
+
+There are two copies of TON blockchain: mainnet and testnet.
+
+_Mainnet_ is the "real" network. It's where actual payments in Toncoin are made. Applications use mainnet by default, and some don't even have an option to use other networks.
+
+The other network is _testnet_, and it is used by TON developers to check that their applications work correctly before deploying them to mainnet. It uses "test coins" that barely have any value.
+
+Each network is split into _workchains_ that can freely interact with each other, but their implementations may differ significantly.
+
+At the moment, there are two workchains: _basechain_ (`0`) for regular use, and a very similar _masterchain_ (`-1`) for TON's internal [bookkeeping](/foundations/system). The masterchain follows mostly the same rules, except that using it is more expensive to limit the amount of traffic that interferes with TON's internals.
+
+To be freely scalable, each workchain is split into _shards_. The number of shards is [determined dynamically](/foundations/shards) based on the current network load. Internally, every shard is implemented as a separate blockchain. Except for increased latency when messages are sent between distant shards, the effect on the user-facing code is minimal.
+
+Physically, the blockchain runs on a network of servers, called [nodes](/ecosystem/node/overview). They interact via the _ADNL_ protocol. The physical layer mostly doesn't affect user-facing applications, as they usually use [servers](/ecosystem/api/overview) that proxy JSON HTTP requests into the ADNL network. The official version of such a proxy server is provided by the _liteserver_ software. There are public instances of liteserver, so developers are not required to host one on their own servers.
+
+ADNL works over [TCP](https://en.wikipedia.org/wiki/Transmission_Control_Protocol), so it's impossible to connect to ADNL from a web page. Browsers [intentionally](https://security.stackexchange.com/q/100827) do not provide an API to send arbitrary TCP traffic. Servers do not have this limitation, as they do not use a browser and can interact with the blockchain over either ADNL or HTTP.
+
+![Statuses overview](/resources/images/states.png)
```

Comment:
why not put this image below? when reading from top to bottom it's not clear what this image is related to from the parts above..